spec/onchain.tex

```diff
@@ -329,7 +387,7 @@ \subsection{Decrement Transaction}\label{sec:decrement-tx}
 
 \noindent The state-machine validator $\nuHead$ is spent with
 $\redeemerHead = (\mathsf{decrement}, \xi, s, m)$, where $\xi$ is a multi-signature of
-the decrement snapshot which authorizes removal of some UTxO, $s$ is the
+the decrement snapshot which authorizes removal of some UTxO $\red{U_\theta}$, $s$ is the
```

Comment:
Actually, we do not need to give this UTxO a variable here (we don't use it in the constraints).

```suggestion
the decrement snapshot which authorizes removal of some UTxO, $s$ is the
```